src/backend/distributed/utils/maintenanced.c

```diff
@@ -644,7 +644,8 @@ CitusMaintenanceDaemonMain(Datum main_arg)
 				 */
 				lastShardCleanTime = GetCurrentTimestamp();
```

Comment:
Should we set this after a failure?

From the comment, it seems okay to set it